pitch.md

```diff
@@ -1,19 +1,85 @@
 them?_
+I am helping students taking economics courses with my DSL. Currently, the 
+experience of students in economics classes is less than ideal. Note-taking is
+done by hand, with both written words and graphs being taken note of, which is
+done simultaneously with listening to lectures. This doesn't sound like a
+terrible experience; however, I believe it can be vastly improved.  Much of the
+frustration is rooted in the fact that lecture slides often contain major points
+or leading questions of a lecture, but don't answer these questions. Rather, the
+answers or explanations are given verbally. It goes without saying that 
+assessments revolve around the answers to the major questions and therefore the
+things _said_ in class. As a result, it's often important to be able to 
+transcribe the things Professors say in class for several minutes at a time. 
+
+Transcribing is much easier on a computer.  Typing removes the pain point of 
+illegible handwriting when trying to keep up with the Professor's lecture, and
+is simply faster.  However, using a computer doesn't allow for easy switching 
+between typing and creating graphs. In fact, creating graphs in general is
+difficult on a computer.  
+
+Thus, if I could help, I would allow students to type notes to transcribe things
+said in class, but also easily create graphs in the same place.  This means that
+by typing some commands, we could create any graph that a student would come 
+accross in their economics classes.  Of course, some would be easier to create
+than others (simple supply-demand graphs, production possibilities curves, etc.),
+but ideally all types of economics graphs would be possible to generate. 
 
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
-
+A DSL is appropriate for our users because of several reasons.  Most importantly,
+I believe that it's appropriate because note-taking involves language. Thus, if 
+we want to improve note-taking, we should do it with some form of language.  We
+need to create some way of creating graphs without interfering too much with
+the flow of taking notes in a human language, so a language versus an app 
+makes a lot of sense in this case.
+
+Furthermore, since we want to improve note taking by moving it from hand-written
+to on a computer, it requires some sort of programming language. However, since
+the problem we're trying to solve, easily creating graphs in the same place you
+take notes, is a very specific domain, we should require as little overhead as
+possible for users to learn the language.  Thus, a DSL makes more sense than a 
+general programming language for our users.  A DSL also makes more sense than
+general graphing libraries that may be available in different languages because
+it'll be much more focused on a specific domain and therefore should be very
+easy to learn, in addition to being able to be run in a text editor or note
+taking app of some sort.
```

Comment:
Totally agree. It will probably also be quicker to use a DSL than a library or general-purpose language, and speed of note-taking is critical for this domain.